node/node.go

```diff
@@ -1060,6 +1067,16 @@ func (n *Node) getLatestValidators() (map[uint32][]*state.ValidatorInfo, map[str
 	return validators, nil, nil
 }
 
+// EncodeAddressPubkey will encode the provided address public key bytes to string
+func (n *Node) EncodeAddressPubkey(pk []byte) string {
+	return n.addressPubkeyConverter.Encode(pk)
+}
+
+// DecodeAddressPubkey will try to decode the provided address public key string
+func (n *Node) DecodeAddressPubkey(pk string) ([]byte, error) {
+	return n.addressPubkeyConverter.Decode(pk)
```

Comment:
I think a nil check for the addressPubkeyConverter is missing

Reply:
added

Reply:
ok